recipes-bsp/barebox/barebox.inc

```diff
@@ -24,13 +24,23 @@ COMPATIBLE_MACHINE = "none"
 
 EXTRA_OEMAKE = "CROSS_COMPILE=${TARGET_PREFIX}"
 
+def find_cfgs(d):
+    sources=src_patches(d, True)
+    sources_list=[]
+    for s in sources:
+        if s.endswith('.cfg'):
+            sources_list.append(s)
+
+    return sources_list
+
 do_configure() {
 	if [ -e ${WORKDIR}/defconfig ]; then
 		cp ${WORKDIR}/defconfig ${S}/.config
 	else
 		bbwarn "No defconfig given"
 	fi
 
+	scripts/kconfig/merge_config.sh -m .config ${@" ".join(find_cfgs(d))}
```

Comment:
Note to self: apparently `merge_config.sh` will still work as a noop even with an empty second parameter.